### ISSUE TITLE
Allow passing of aria-label for Accessibility

### ISSUE THREAD

**Issue by megleighc** [Availity/react-block-ui#66]
I am unable to pass down an `aria-label` attribute to add better context to the loading indicator.  Currently, the component is announcing "dot dot dot" when using a screen reader instead of something more informative. Having the ability to pass this attribute will help teams be able to provide more informative labels to the loading indicators in their components. 

The issue, along with a screen shot of ANDI showing what I hear using a screen reader, can be found in our Availity SO [here](https://stackoverflow.com/c/availity/questions/681).

**Comment by megleighc** [comment]
This has been resolved by adding the ability to pass down an `ariaLabel` prop to the BlockUI component.
